app/models/api_key_scope.rb

```diff
@@ -37,7 +37,10 @@ def default_mappings
           bookmarks: { actions: %w[users#bookmarks], params: %i[username] },
           sync_sso: { actions: %w[admin/users#sync_sso], params: %i[sso sig] },
           show: { actions: %w[users#show], params: %i[username external_id] },
-          check_emails: { actions: %w[users#check_emails], params: %i[username] }
+          check_emails: { actions: %w[users#check_emails], params: %i[username] },
+          update: { actions: %w[users#update], params: %i[user_id] },
```

Comment:
I see other scopes use the `username` param. Is there any difference between that and `user_id`?

Reply:
Oh you're right... that should be username
```suggestion
          update: { actions: %w[users#update], params: %i[username] },
```